src/main/kotlin/com/terraformation/backend/admin/AdminVotersController.kt

```diff
@@ -0,0 +1,251 @@
+package com.terraformation.backend.admin
+
+import com.terraformation.backend.accelerator.db.DefaultVoterStore
+import com.terraformation.backend.accelerator.db.VoteStore
+import com.terraformation.backend.api.RequireGlobalRole
+import com.terraformation.backend.customer.db.UserStore
+import com.terraformation.backend.customer.model.requirePermissions
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.VoteOption
+import com.terraformation.backend.db.accelerator.tables.references.COHORTS
+import com.terraformation.backend.db.accelerator.tables.references.PARTICIPANTS
+import com.terraformation.backend.db.accelerator.tables.references.PROJECT_VOTE_DECISIONS
+import com.terraformation.backend.db.default_schema.GlobalRole
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.UserId
+import com.terraformation.backend.db.default_schema.tables.references.PROJECTS
+import com.terraformation.backend.log.perClassLogger
+import org.jooq.DSLContext
+import org.jooq.Record
+import org.springframework.stereotype.Controller
+import org.springframework.ui.Model
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.servlet.mvc.support.RedirectAttributes
+
+@Controller
+@RequestMapping("/admin")
+@RequireGlobalRole([GlobalRole.SuperAdmin])
+@Validated
+class AdminVotersController(
+    private val defaultVoterStore: DefaultVoterStore,
+    private val dslContext: DSLContext,
+    private val userStore: UserStore,
+    private val voteStore: VoteStore,
+) {
+  private val log = perClassLogger()
+
+  @GetMapping("/voters")
+  fun getVoters(
+      @RequestParam email: String?,
+      model: Model,
+      redirectAttributes: RedirectAttributes,
+  ): String {
+    requirePermissions { updateDefaultVoters() }
+
+    val userIdForEmail = email?.let { userStore.fetchByEmail(it)?.userId }
+
+    val projects = getAcceleratorProjects()
+    val projectVoters =
```

Comment:
Maybe throw an explicit type on here just for clarity. I think it's `Map<ProjectId, Set<UserId>>` but figuring that out takes a little effort.